### ISSUE TITLE
MacOS Build Errors

### ISSUE THREAD

**Issue by ktoksoz** [f4exb/sdrangel#430]
When building in macOS Catalina (10.15) I encountered with following errors:

```
/Users/koray/Projects/HAM/sdr/sdrangel/sdrangel/sdrbase/ambe/ambeengine.cpp:141:26: error: variable has incomplete type 'struct serial_struct'
    struct serial_struct serinfo;
                         ^
/Users/koray/Projects/HAM/sdr/sdrangel/sdrangel/sdrbase/ambe/ambeengine.cpp:141:12: note: forward declaration of 'serial_struct'
    struct serial_struct serinfo;
           ^
/Users/koray/Projects/HAM/sdr/sdrangel/sdrangel/sdrbase/ambe/ambeengine.cpp:154:27: error: use of undeclared identifier 'TIOCGSERIAL'
            if (ioctl(fd, TIOCGSERIAL, &serinfo) == 0)
                          ^
/Users/koray/Projects/HAM/sdr/sdrangel/sdrangel/sdrbase/ambe/ambeengine.cpp:157:37: error: use of undeclared identifier 'PORT_UNKNOWN'
                if (serinfo.type != PORT_UNKNOWN) {
                                    ^

```
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/1962363/66718134-21b23200-ede9-11e9-8da7-f96ea181f862.png">

**Comment by rgetz** [comment]
The readme states:

The platform of choice to run SDRangel is definitely Linux

Which means - most (all) the developers use Linux and don’t have access to MAC. Well formatted patches are always reviewed if you have the time to look into things in more detail.

**Comment by f4exb** [comment]
That's perfectly right however at some point before the redesign of the AMBE engine it used to compile for Mac so I will try to restore the situation by skipping the "linuxities" introduced if compiled with `__APPLE__`. 

Now on dev branch there should be a version that compiles with Mac.

**Comment by f4exb** [comment]
Should be fixed in v4.11.11
